packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -0,0 +1,314 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+    useAccountStatus,
+    useActiveTradingAccount,
+    useAvailableMT5Accounts,
+    useCreateMT5Account,
+    useMT5AccountsList,
+    useSettings,
+    useTradingPlatformPasswordChange,
+} from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { Button, useBreakpoint } from '@deriv/quill-design';
+import { ActionScreen, ButtonGroup, Dialog, ModalStepWrapper, SentEmailContent } from '../../../../components';
+import MT5PasswordIcon from '../../../../public/images/ic-mt5-password.svg';
+import { TMarketTypes, TPlatforms } from '../../../../types';
+import { validPassword } from '../../../../utils/password';
+import {
+    Category,
+    companyNamesAndUrls,
+    MarketType,
+    MarketTypeDetails,
+    PlatformDetails,
+    QueryStatus,
+} from '../../constants';
+import { CFDSuccess, CreatePassword, EnterPassword } from '../../screens';
+import { Jurisdiction } from '../../screens/CFDCompareAccounts/constants';
+
+type TMT5PasswordModalProps = {
+    marketType: TMarketTypes.SortedMT5Accounts;
+    platform: TPlatforms.All;
+};
+
+const MT5PasswordModal: FC<TMT5PasswordModalProps> = ({ marketType, platform }) => {
```

Comment:
Let's not normalise using `FC` 😅  is not necessary and it just makes you import more stuff from `React`. Just keep it as it is
```suggestion
const MT5PasswordModal = ({ marketType, platform }: TMT5PasswordModalProps) => {
```